src/browser/webview/mattermost.js

```diff
@@ -3,16 +3,13 @@
 // See LICENSE.txt for license information.
 'use strict';
 
-import {ipcRenderer, webFrame} from 'electron';
+/* eslint-disable no-magic-numbers */
```

Comment:
is this really needed? can we instead fix the error?

Reply:
There are a number of these all through the Destkop code that need to be looked at. I added this in the particular case to temporarily stop vscode from complaining while I was working on the notifications stuff. I'll make sure there is a ticket in the backlog to address along with a ticket to convert bind to arrow functions.

Reply:
https://mattermost.atlassian.net/browse/MM-18755